### ISSUE TITLE
Accessibility issue - checkbox labelling

### ISSUE THREAD

**Issue by rmbielby** [dfe-analytical-services/CSC_outcomes_and_enablers#115]
**Describe the bug**

Checkbox form elements do not have appropriate labels.

**To Reproduce**
Run aXetools accessibility test on dashboard

**Expected behaviour**
Checkboxes should have correct labels. Could be a case of switching to shinyGovstyle if shinyWidgets has been used.
